astro_2.0/components/PageLayout/PageLayout.module.scss

```diff
@@ -7,7 +7,7 @@
   width: 100%;
 
   &.whiteBackground {
-    background-color: var(--color-white);
+    //background-color: var(--color-white);
```

Comment:
Do we still need this className?

Reply:
It is still under development - I left it as a placeholder as on screens we have different colors on different screens